test/integration/test_backend_serialization.py

```diff
@@ -76,7 +76,8 @@ def test_backend_properties(self, service):
         for backend in backends:
             with self.subTest(backend=backend):
                 properties = backend.properties()
-                self._verify_data(properties.to_dict(), good_keys)
+                if properties:
+                    self._verify_data(properties.to_dict(), good_keys)
```

Comment:
Adding this check here because `fake_backend1` is no longer a simulator and causes this line to error